data/human/news.txt

```diff
@@ -111,6 +113,19 @@ news "human merchant in human space"
 			"I was doing some jobs in the Core, when an enormous alien warship jumped in and attacked! People say I'm crazy, but I'm lucky to have gotten out of there with my life!"
 			"The Syndicate tries to keep alien attacks under wraps, but if you've been around as long as I have, you'll see them eventually."
 			"I was working a Syndicate defense job once, expecting to only deal with some pirates. Instead, I had to drive off an alien warship that jumped in same way the Quarg do! They made us sign a non-disclosure agreement, so don't tell anyone I said that."
+			"I once got hired by someone in search for his true love; he didn't know which ship she left in, so we travelled to dozens of planets to search for her, until he ran out of credits."
+			"I once had a passenger that returned to a spaceport where she met her true love. I wonder if they ever got together again."
+			"I once put 50,000 rifles and 150,000 grenades into my armory. One grenade got melted into the floor due to the weight of all the other weaponry on top of it and I could no longer sell that grenade."
+			"I felt lonely, so I participated in Romantic Matchmaking on a Syndicate planet, but that was a big scam! They never asked anything about the type of partner I was looking for, they only asked questions about what I wanted to buy in the next 10 years."
+			"I once joined Romantic Matchmaking organized by the Syndicate, but they couldn't tell me anything about my true love due to privacy concerns. Can you believe that?"
+			"Romantic Matchmaking as organized by the Syndicate is garbage; the Syndicate laws don't allow differentiating on gender, so there is a 50% chance that the match they find for you is the wrong gender."
```

Comment:
Isn't the Syndicate implied to be sexist in lore?

Reply:
Yes, they are. I removed the part about syndicate law. (And left it unspecified why those matchmaking programs don't differentiate on gender.)